test/builtin/files_io/test_files.py

```diff
@@ -102,6 +102,8 @@ def test_close():
 @pytest.mark.parametrize(
     ("str_expr", "msgs", "str_expected", "fail_msg"),
     [
+        (None, None, None, None),  # Reset the session and set the working
```

Comment:
It is not obvious to me why this should work. Couldn't this be done as test setup instead?